android/scripts/make_rust.sh

```diff
@@ -1,52 +1,91 @@
 #!/bin/bash
 
 # Run from repo's root folder (relative to Gradle build file. TODO do this in Gradle)
-cd ../..
-echo "Running make_rust.sh from $(pwd)"
+
+if [ -d "android/core/gradle" ]; then
+    echo "Starting..."
+else
+    echo "Error: this script must be run in project root"
+    exit 5
+fi
```

Comment:
Doesn't this fail when started from Gradle (in the Android core app)?

Reply:
Likely. I want to make a separate PR for gradle support (or cargo-make as mentioned in #121)

Reply:
Fixed: script logic is anchored to its parent folder